exchanges/btse/btse_wrapper.go

```diff
@@ -972,7 +972,7 @@ func (b *BTSE) FormatExchangeKlineInterval(in kline.Interval) string {
 
 // GetHistoricCandles returns candles between a time period for a set time interval
 func (b *BTSE) GetHistoricCandles(ctx context.Context, pair currency.Pair, a asset.Item, interval kline.Interval, start, end time.Time) (*kline.Item, error) {
-	req, err := b.GetKlineRequest(pair, a, interval, start, end)
+	req, err := b.GetKlineRequest(pair, a, interval, start, end, false)
```

Comment:
I'm not sure what's gone wrong here:
```
% go run . gethistoriccandles --exchange="btse" --pair="btc-usd" --asset="spot" -r="300"
2023/04/13 09:33:21 rpc error: code = Unknown desc = BTSE unsuccessful HTTP status code: 400 raw response: {"code":400,"msg":"BADREQUEST: resolution too small for the requested time range. Records returned exceeds 300","success":false,"time":1681342401899,"data":[]}
```

Reply:
End time is inclusive, will add comment and change. Nice spotting!